### DISCUSSION TITLE
Forge not compiling Solidity contract - "Nothing to compile" error

### DISCUSSION THREAD

**Discussion by Airikr02** [Cyfrin/foundry-full-course-cu#481] [Q&A] [answered]
Hello everyone,

I've been working on a Solidity project and I'm currently facing an issue where forge create doesn't compile my contract, returning a "Nothing to compile" error message.

Could someone please help me understand why my contract isn't being compiled and how I might be able to resolve this issue? Any advice or guidance would be greatly appreciated.

Thank you!

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/78366349/f12152bc-6c12-4a84-b426-8f5217e700e3)

**Comment by shikhar229169** [comment, marked as answer]
Actually in the command line you are not in the correct directory.
You need to be in the folder ```foundry-simple-storage-f23```, and you can do it by:
```
cd foundry-f23/foundry-simple-storage-f23
```

**Reply by Airikr02** [reply]
Ahhh thanks mate
